core-plugins/src/test/java/co/cask/hydrator/plugin/batch/action/HDFSFileMoveActionTestRun.java

```diff
@@ -213,19 +195,166 @@ public void testInvalidDestHDFSAction() throws Exception {
     manager.start();
     manager.waitForFinish(3, TimeUnit.MINUTES);
 
-//    List<RunRecord> history = appManager.getHistory(new Id.Program(appId, ProgramType.WORKFLOW, SmartWorkflow.NAME),
-//                                                    ProgramRunStatus.FAILED);
-//    Assert.assertTrue(history.size() == 1);
-//
-//    Map<String, WorkflowNodeStateDetail> nodesInFailedProgram =
-//      manager.getWorkflowNodeStates(history.get(0).getPid());
-//    Assert.assertTrue(nodesInFailedProgram.size() == 1);
-//
-//    //check that HDFSFileMoveAction node failed
-//    Assert.assertTrue(nodesInFailedProgram.values().iterator().next().getNodeStatus().equals(NodeStatus.FAILED));
-
     Assert.assertTrue(fileSystem.exists(new Path(outputDir.toUri().toString() + "/source/test.txt")));
     Assert.assertTrue(fileSystem.exists(new Path(outputDir.toUri().toString() + "/source/test2.txt")));
     Assert.assertTrue(fileSystem.exists(new Path(outputDir.toUri().toString() + "/source/test.json")));
   }
+
+  @Test
+  public void testHDFSDeleteAction() throws Exception {
+
+    Path outputDir = dfsCluster.getFileSystem().getHomeDirectory();
+
+    fileSystem.mkdirs(new Path("dir"));
```

Comment:
is the same directory used in all tests?  It doesn't have any affect right now, but it's better to use a different directory per test, or make sure you clean up after the test finishes to make sure tests are isolated from each other.